src/views/CountryNsOverviewActivities/index.tsx

```diff
@@ -1,16 +1,527 @@
+        ),
+        createDateRangeColumn<Project, number>(
+            'startDate',
+            strings.nSStartDate,
+            (item) => ({
+                startDate: item.start_date,
+                endDate: item.end_date,
+            }),
+        ),
+        createStringColumn<Project, number>(
+            'disasterType',
+            strings.nSTableDisasterType,
+            (item) => item.dtype_detail?.name,
+        ),
+        createStringColumn<Project, number>(
+            'sector',
+            strings.nSTableSector,
+            (item) => item.primary_sector_display,
+        ),
+        createStringColumn<Project, number>(
+            'name',
+            strings.nSTableProjectName,
+            (item) => item.name,
+        ),
+
+        createNumberColumn<Project, number>(
+            'budget',
+            strings.nSTableTotalBudget,
+            (item) => item.budget_amount,
+            undefined,
+        ),
+        createNumberColumn<Project, number>(
+            'peopleTargeted',
+            strings.nSTablePeopleTargeted,
+            (item) => item.target_total,
+            undefined,
+        ),
+        createNumberColumn<Project, number>(
+            'peopleReached',
+            strings.nSTablePeopleReached,
+            (item) => item.reached_total,
+            undefined,
+        ),
+        createStringColumn<Project, number>(
+            'contact',
+            strings.nSContactPerson,
+            (item) => (
+                `${item.reporting_ns_contact_name}
+                ,
+                ${item.reporting_ns_contact_email}`
+            ),
+        ),
+        createElementColumn<Project, number, ProjectActionsProps>(
+            'actions',
+            '',
+            ProjectActions,
+            (_, project) => ({
+                onProjectDeletionSuccess: reTriggerProjectListRequest,
+                className: styles.actions,
+                project,
+            }),
+        ),
+    ]), [
+        reTriggerProjectListRequest,
+        strings.nSTableCountries,
+        strings.nSTableProjectName,
+        strings.nSTableSector,
+        strings.nSTableTotalBudget,
+        strings.nSTableDisasterType,
+        strings.nSTablePeopleTargeted,
+        strings.nSTablePeopleReached,
+        strings.nSContactPerson,
+        strings.nSStartDate,
+    ]);
+
+    const countryIdList = Object.keys(countryGroupedProjects);
+
+    const [
+        pendingExport,
+        progress,
+        triggerExportStart,
+    ] = useRecursiveCsvExport({
+        onFailure: () => {
+            alert.show(
+                strings.nSFailedToCreateExport,
+                { variant: 'danger' },
+            );
+        },
+        onSuccess: (data) => {
+            const unparseData = Papa.unparse(data);
+            const blob = new Blob(
+                [unparseData],
+                { type: 'text/csv' },
+            );
+            saveAs(blob, `${countryResponse?.name}-data-export.csv`);
```

Comment:
Let's add more context to the file name and also .
```suggestion
            saveAs(blob, `${countryResponse?.society_name}-international-works.csv`);
```